### PR TITLE
Jb57230

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
Just the very last commit belongs to this PR

File: sdk-setup/src/sdk-manage [hunk 1]
```diff
@@ -489,12 +489,13 @@ get_all_or_selected_of_type() {
...
     fi
 
-    # Exclude snapshots
+    # Exclude snapshots unless required explicitly
     if [[ $type == target ]]; then
         local objects_=("${objects[@]}")
         objects=()
         for object in "${objects_[@]}"; do
-            target_is_snapshot "$object" || objects+=("$object")
+            ! target_is_snapshot "$object" || ! [[ $get_all ]] || continue
```
Comment (vigejolla) [inline sdk-setup/src/sdk-manage:497]: This makes my brain hurt. But I *think* the logic is correct.

**Review by vigejolla — APPROVED** [review]
LGTM

File: sdk-setup/src/sdk-manage (hunk 1 above)
Comment (martyone) [reply sdk-setup/src/sdk-manage:497]: Whenever you see `!A || B`, read it as `A -> B` :)

Comment (vigejolla) [reply sdk-setup/src/sdk-manage:497]: Yes, I know. Still having two of those in a row is too much for me, at least on a friday afternoon :)

**Pull request merged**